hera_mc/observations.py

```diff
@@ -58,7 +66,7 @@ def length(self):
         return self.stoptime - self.starttime
 
     @classmethod
-    def create(cls, starttime, stoptime, obsid, hera_cofa):
+    def create(cls, starttime, stoptime, obsid, hera_cofa, tag):
```

Comment:
Do we want to explicitly check that `tag` is one of the suggested columns? I don't think we want this to be an actual `enum`, but it might be good to enforce it's in one of the delineated categories.

Reply:
I wasn't sure. I'm happy to add some checking on that. If it doesn't match, do we want to error or just warn?